test/evergreen_develop.yml

```diff
@@ -178,19 +157,9 @@ buildvariants:
   run_on:
   - ubuntu2004-test
   expansions:
-    configure_env_vars: PATH=/opt/mongodbtoolchain/v4/bin:$PATH
-    compile_env_vars: PATH=/opt/mongodbtoolchain/v4/bin:$PATH
-    test_env_vars:
-      PATH=/opt/mongodbtoolchain/v4/bin:$PATH
-      WT_TOPDIR=$(git rev-parse --show-toplevel)
-      WT_BUILDDIR=$WT_TOPDIR/cmake_build
-      LD_LIBRARY_PATH=$WT_BUILDDIR:$WT_TOPDIR/TCMALLOC_LIB/lib
-      LD_PRELOAD=$WT_TOPDIR/TCMALLOC_LIB/lib/libtcmalloc.so
     CMAKE_PREFIX_PATH: -DCMAKE_PREFIX_PATH="$(pwd)/TCMALLOC_LIB"
-    python_binary: '/opt/mongodbtoolchain/v4/bin/python3'
     smp_command: -j $(echo "`grep -c ^processor /proc/cpuinfo` * 2" | bc)
     cmake_generator: Ninja
```

Comment:
Do we need to specify `cmake_generator`? It seems that the default value `Ninja` is used.

Reply:
I agree, do you think we should just default to Ninja instead? I can understand it can be used for readability to acknowledge a variant is using Ninja vs Make file. @lukech what do you think?

Reply:
I'm okay with `cmake_generator` setting line being removed (if the value is "Ninja" - the default).

Reply:
[3856cab](https://github.com/wiredtiger/wiredtiger/pull/9533/commits/3856cab1ed5f29fae68eca15064a0b18d9609cfb)